hafnian/_hafnian.py

```diff
@@ -174,7 +175,10 @@ def hafnian(
 
 
 def hafnian_repeated(A, rpt, mu=None, loop=False, tol=1e-12):
-    r"""Returns the hafnian of matrix with repeated rows/columns.
+    r"""
+    THIS NEEDS TO BE WRITTEN
+
+    Returns the hafnian of matrix with repeated rows/columns.
```

Comment:
Is this a copy/paste mistake :slightly_smiling_face: Probably should undo this change.

Reply:
It is. Thanks for catching it up!